libs/experimental/Makefile

```diff
@@ -15,6 +15,9 @@ tests:
 test_watch:
 	poetry run ptw --now . -- tests/unit_tests
 
+extended_tests:
```

Comment:
It should be in other PR, also likely CONTRIBUTIONS or some docs needs to be updated.